superset/models/sql_lab.py

```diff
@@ -94,6 +94,8 @@ class Query(Model, ExtraJSONMixin):
         sqla.Index('ti_user_id_changed_on', user_id, changed_on),
     )
 
+    # TODO: Clean up/fix/refactor limit_reached and limit_used. limit_used defaults
```

Comment:
Agreed looks like we can drop both now. I'm guessing they got orphaned with the back and forth around limits